pkg/ccl/backupccl/backup_test.go

```diff
@@ -6255,7 +6265,9 @@ func TestPaginatedBackupTenant(t *testing.T) {
 	withoutTS := hlc.Timestamp{}
 
 	const numAccounts = 1
-	serverArgs := base.TestServerArgs{Knobs: base.TestingKnobs{JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals()}}
+	serverArgs := base.TestServerArgs{
+		Knobs:                    base.TestingKnobs{JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals()},
+		DisableDefaultTestTenant: true}
```

Comment:
ditto as above, a short comment on why this is true since you've already spent the time investigating why.